docs/install-commcare-sync.md

```diff
@@ -1,65 +1,99 @@
-Production Environments
-=======================
+Set up CommCare Sync
+=====================
+This page outlines the process you need to follow in order to set up a CommCare Sync instance on a machine.
 
-This page describes creating and managing production inventory files.
-For details on deploying a production environment, see the [deployment page](/deployment).
+## Install Ansible
 
-### Directory overview
+From the [Ansible Installation Guide](https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html#installing-ansible-on-ubuntu),
+run the following on your local/control machine
 
-Production environments live in the `inventories/` folder.
+```bash
+sudo apt-add-repository ppa:ansible/ansible
+sudo apt-get update
+sudo apt-get install ansible
+```
+
+## Install CommCare Sync
+
+### Choose a location for the control
+
+While it is possible to run the ansible playbooks from anywhere, 
+it is recommended to run them *on the server you are setting up*.
+This will ensure consistency and also streamline the install process.
+
+These instructions assume a set up where the ansible control and playbooks are run on the server being set up.
 
-To add a new production environment you can follow the steps below, replacing `myproject` with your project name.
+### Set up user accounts
```

Comment:
Is there value in creating a separate ansible user in dev? is that the workflow we are now recommending?